docs/StandardLibrary.md

```diff
@@ -0,0 +1,334 @@
+# Flux Standard Library Design
+
+The Flux standard library will define the set of functions, constants, and other
+values that are available out of the box with Flux, providing the functionality that is
+most frequently used.
+
+Some functions in the standard library will be available without qualifying
+them with a namespace.  These will be the most common functions, to help Flux
+code be more concise.  Such functions are said to be in the "top level package."
+
+The remaining functions in the standard library will be divided up into
+discrete packages that are organized by topic.
+
+What's presented here is just a departure point for further discussion.
+
+## The Top Level Package
+
+Those functions that provide functionality that is very frequently
+used and should not need to be qualified with a namespace in Flux code.
+(Consider also the concept of "prelude")
+
+### Transformations
+Should some of these be in the `math` package instead?  `pearsonr` for example.
```

Comment:
I think the math package should be restricted to working with raw numeric types. As such pearsonr would not be part of the math package, because it deals with tables and streams.

That said pearsonr could be placed into a different package, perhaps a "stats" package?

Reply:
I like this idea but I predict some confusion as to where some stream functions should live, since `mean` for example is a statistic but is so common that it may be reasonably expected to be top level.

One could imagine having it live in both places:
```
// Top level package
mean = stats.mean
```